docker/sublime/Dockerfile

```
@@ -0,0 +1,12 @@
+
+FROM demisto/python3:3.10.14.97608
```

```suggestion
FROM demisto/python3:3.10.14.98471
```
We encourge to use the latest docker image.